src/main/java/com/github/thundersphun/bcf/funnel/FunnelBlockEntityRenderer.java

```diff
@@ -43,7 +45,7 @@ public void render(FunnelBlockEntity entity, float tickDelta, MatrixStack matric
 		}
 
 		if (entity.getWaterFullness() > 0) {
-			var waterColor = world.getBiome(entity.getPos()).getWaterColor();
+			var waterColor = BuiltinRegistries.BIOME.get(BiomeKeys.OCEAN).getWaterColor();
```

Comment:
an easy fix i found is this:
`var waterColor = BuiltinRegistries.BIOME.get(world.getBiome(entity.getPos()).getKey().orElse(null)).getWaterColor();`